README.md

```diff
@@ -1,5 +1,8 @@
 # kempctl
 
+[![Go Report Card](https://goreportcard.com/badge/github.com/giantswarm/kempctl)](https://goreportcard.com/report/github.com/giantswarm/kempctl) [![Godoc](https://godoc.org/github.com/giantswarm/kempctl?status.svg)](http://godoc.org/github.com/giantswarm/kempctl)
```

Comment:
Docker is the only one missing, because I don't want to sort out the Hub stuff yet - see https://github.com/giantswarm/kempctl/issues/9 for tracking